docs/8-call-status.md

```diff
@@ -0,0 +1,75 @@
+# Nexmo In-App Call States 
+
+In this quick start we are going to explain call statuses for _outgoing calls_ like 
+ started, ringing, answered, rejected, busy, unanswered, timeout, failed, complete, machine. Since everyone is familiar with the phone, these concepts are pretty straightforward. We won't waste anytime explaining them but we will briefly mention what they signify. After explaining what they signify we  will cover how to subscribe to changes in state in an app that has implemented that Nexmo In-App Calling features with its iOS SDK. 🙌
+
+# What do the call states signify?
+
+Every Nexmo In-App call has a status value that signifies the current state of a call. Since call status is subject to change in real time, there must be a way to track these changes. Nexmo In-App Calling States help us keep track of these changes! 🎉 🎉 🎉 And there a lot of states! 
+
+### States
+
+Here is a handy dandy table with a brief list of all states: 
+
+- **started** : Platform has initiated the call
+- **ringing** : Caller has joined and pending for at least one more to join
+- **answered** : The user has answered your call
+- **rejected** : The call was rejected
+- **busy**: Caller is busy
+- **unanswered**: The call was canceled from the called before an answer
+- **timeout**: User did not answer your call with ringing_timer seconds
```

Comment:
...within the set time of X seconds.. cannot recall what this number is. there is a default mac value of seconds coming from CS

Reply:
@mariatechmaniac I could not find this value. Are you certain it won't change?